libs/blocks/marquee/marquee.css

```diff
@@ -1,9 +1,45 @@
+.marquee.large .foreground.container .text {
+  margin: var(--spacing-m) auto;
+  width: var(--grid-container-width);
+}
+
+.marquee.large .container {
+  width: 100%;
+  padding: 0;
+  gap:0;
+}
+
+.marquee .icon-area.icon-area-multiple > :not(:last-child) img {
+  margin: 0 var(--spacing-s) 0 0;
+}
+
+.marquee.large .text .heading-xxl {
+  margin-bottom: var(--spacing-s);
+}
+
+.marquee.large .text .body-xl:not(.supplemental-text){
+  margin-bottom: 0;
+}
+
+.marquee.large .body-xl .icon {
+  margin: var(--spacing-xs) var(--spacing-xs) var(--spacing-xs) 0;
```

Comment:
If this being used somewhere? icon-area does not have a body-xl attached to its classlist and I could not find .icon class on any element. Am I missing something?

Reply:
> If this being used somewhere? icon-area does not have a body-xl attached to its classlist and I could not find .icon class on any element. Am I missing something or does it support library icons only and not any svg link?

This is implementing the use case when the body has 18px icons. We are currently waiting for the consonant team to confirm whether the icons will be added from the library or does it support svg links.
Thank you.